build.sbt

```diff
@@ -12,7 +12,7 @@ lazy val `sbt-freestyle` = project
   .in(file("."))
   .settings(name := "sbt-freestyle")
   .settings(Seq(
-    addSbtPlugin("com.47deg" % "sbt-org-policies" % "0.7.5"),
+    addSbtPlugin("com.47deg" % "sbt-org-policies" % "0.7.8"),
```

Comment:
Can you switch this to `%("sbt-org-policies", true)`?
One less place to change a version number in the future.